python/rmm/_lib/device_buffer.pyx

```diff
@@ -34,6 +34,18 @@ cdef class DeviceBuffer:
     def size(self):
         return self.c_obj.get()[0].size()
 
+    @property
+    def __cuda_array_interface__(self):
+        cdef dict intf = {
+            "data": self.data(),
+            "shape": (self.size(),),
+            "strides": None,
+            "typestr": "|u1",
+            "descr": [("", "|u1")],
```

Comment:
This is optional so we don't have to define it given we don't have a complex type here.

Reply:
Sure. Happy to drop this.

Reply:
Dropped.